packages/demo-app/src/examples/DatePickerExample.tsx

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2020 Palantir Technologies, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+
+import * as React from "react";
+
+import { Classes } from "@blueprintjs/core";
+import { DatePicker } from "@blueprintjs/datetime";
+
+import { ExampleCard } from "./ExampleCard";
+
+const WIDTH = 300;
+export const DatePickerExample = React.memo(() => {
```

Comment:
why memo?

Reply:
oh i'm just used to using memoized functional components since that's the norm in most of Nucleus - happy to use regular functional components though

Reply:
hmm ok, I haven't really seen it used before but I don't mind it as long as the component works as intended... fine to leave as-is